src/layout/section/itemsSection/index.tsx

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components'
```

Comment:
should not be there :)

Reply:
Not part of this PR (created via the rebase) ;)